pkg/sql/pgwire/pgwire_test.go

```diff
@@ -1956,3 +1957,31 @@ func TestFailPrepareFailsTxn(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestOverlyLargeRequest(t *testing.T) {
```

Comment:
any reason this you chose to implement a new test method instead of extending `TestConnMessageTooBig`?

Reply:
ah nope! i had copied this test that jordan wrote and forgot i had it committed.
 i think i should modify `TestConnMessageTooBig` so it uses `postLongStrAction` for the prepared statements.